introduction/from_flat_php_to_symfony2.rst

```diff
@@ -717,14 +717,14 @@ And rewriting ``layout.html.php`` template in Twig would look like this:
 
 Twig is well-supported in Symfony. And while PHP templates will always
 be supported in Symfony, the many advantages of Twig will continue to
-be discussed. For more information, see the :doc:`templating chapter </templating>`.
+be discussed. For more information, see the :doc:`templating article </templating>`.
 
 Where Symfony Delivers
 ----------------------
 
-In the upcoming chapters, you'll learn more about how each piece of Symfony
-works and how you can organize your project. For now, celebrate at how migrating
-the blog from flat PHP to Symfony has improved life:
+In the rest of documentation articles, you'll learn more about how each piece of
```

Comment:
It could be valid ... but we're having some troubles with articles that say "previous articles", "next articles", etc.  The docs now follow a non-linear structure, so it's better to say things like "the rest of articles", etc.